index.js

```diff
@@ -181,4 +166,22 @@ module.exports = {
       };
     }), 'browser-fetch');
   },
+
+  _findApp() {
+    if (typeof this._findHost === 'function') {
+      return this._findHost();
+    } else {
+      // Otherwise, we'll use this implementation borrowed from the _findHost()
+      // method in ember-cli.
+      // Keep iterating upward until we don't have a grandparent.
+      // Has to do this grandparent check because at some point we hit the project.
+      let app;
+      let current = this;
+      do {
+         app = current.app || this;
+      } while (current.parent && current.parent.parent && (current = current.parent));
+
+      return app;
```

Comment:
Since this functionality is in cli > 2.17 an this addon uses at least 3.5, seems like we can remove this?

Reply:
But given the current version of this addon tests against 3.5, we dont really support < 2.17...

Reply:
We lack a way to test against different cli versions
https://github.com/tomdale/ember-cli-addon-tests/issues/25